ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddMissingSeqFixTest.fs

```diff
@@ -0,0 +1,20 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type AddMissingSeqFixTest() =
+    inherit FSharpQuickFixTestBase<AddMissingSeqFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/addMissingSeqFix"
+
+    [<Test>] member x.``FS0740 Adds missing seq before sequential`` () = x.DoNamedTest()
```

Comment:
Could you also add more cases where this quick fix is applied to some nested expression?

```fsharp
for _ in {1; 2} do
    ()
```

```fsharp
let _ =
    { 1; 2 }
```